### ISSUE TITLE
Will you add a bow punch function like on the 1.8? very useful it is to create 1.8 servers such as: MegaDrop etc

### ISSUE THREAD

**Issue by OMEREQQ** [kernitus/BukkitOldCombatMechanics#721]
## Proposed solution

Will you add a bow punch function like on the 1.8? very useful it is to create 1.8 servers such as: MegaDrop etc

**Comment by kernitus** [comment]
Are you talking about the enchantment?

**Comment by Xyness** [comment]
I think he's talking about the bow boosting, since 1.9 you can not punch yourself with the bow

**Comment by kernitus** [comment]
So you would want the opposite of the `disable-bow-boost` module?

**Comment by Astrotix** [comment]
Hey, you can hit yourself if you seed arrow on the sky, but you can't do the mechanic run bow boosting.

**Comment by simulatorp53-ai** [comment]
> So you would want the opposite of the `disable-bow-boost` module?

Yeah, I mean the bow mechanics from version 1.8 so you could shoot yourself with an arrow while running and looking straight ahead like on the old servers. Could you manage to add that?
